pkg/controller/manager/sync_action.go

```diff
@@ -10,13 +10,15 @@ import (
 	"github.com/pkg/errors"
 	"github.com/scylladb/go-set/strset"
 	"github.com/scylladb/scylla-operator/pkg/api/scylla/v1"
-	"github.com/scylladb/scylla-operator/pkg/mermaidclient"
+	"github.com/scylladb/scylla-operator/pkg/managerclient"
+	"github.com/scylladb/scylla-operator/pkg/managerclient/model"
+	"github.com/scylladb/scylla-operator/pkg/managerclient/util"
 	"github.com/scylladb/scylla-operator/pkg/naming"
 	"github.com/scylladb/scylla-operator/pkg/util/uuid"
 )
 
 type state struct {
-	Clusters    []*mermaidclient.Cluster
+	Clusters    []*model.Cluster
```

Comment:
Should we keep the manual types at the same level / in the same package as the manual client. @zimnx wdyt?

Reply:
I've changed the file structure so that managerclient package has all the necessary things.